libraries/Microsoft.Bot.Connector/Authentication/GovernmentChannelValidation.cs

```diff
@@ -41,15 +41,15 @@ public sealed class GovernmentChannelValidation
         /// setup and teardown, so a shared HttpClient is recommended.</param>
         /// <param name="channelId">The ID of the channel to validate.</param>
         /// <returns>ClaimsIdentity.</returns>
-        public static async Task<ClaimsIdentity> AuthenticateChannelToken(string authHeader, ICredentialProvider credentials, string serviceUrl, HttpClient httpClient, string channelId)
+        public static async Task<ClaimsIdentity> AuthenticateChannelToken(string authHeader, ICredentialProvider credentials, string serviceUrl, HttpClient httpClient, string channelId, AuthenticationConfiguration authConfig = null)
         {
             var tokenExtractor = new JwtTokenExtractor(
                 httpClient,
                 ToBotFromGovernmentChannelTokenValidationParameters,
                 OpenIdMetadataUrl,
                 AuthenticationConstants.AllowedSigningAlgorithms);
 
-            var identity = await tokenExtractor.GetIdentityAsync(authHeader, channelId).ConfigureAwait(false);
+            var identity = await tokenExtractor.GetIdentityAsync(authHeader, channelId, authConfig?.RequiredEndorsements).ConfigureAwait(false);
 
```

Comment:
Null programming is bad.

Reply:
Not necessarily always, but in this case it is easy to remove :)